artifacts/debian/debian.go

```diff
@@ -111,7 +111,8 @@ func (d *debian) getBuildData(jsonURL string) (additionalTags []string, checksum
 }
 
 func (d *debian) Inspect() (*api.ArtifactDetails, error) {
-	if !strings.HasPrefix(d.Version, "11") && !strings.HasPrefix(d.Version, "12") {
+	if !strings.HasPrefix(d.Version, "11") && !strings.HasPrefix(d.Version, "12") &&
```

Comment:
I wonder if it can be done a bit more elegant, but in the end it does its job.

Reply:
Done. Please, take a look!